docs/run/nodes/nodes.md

```diff
@@ -35,7 +35,7 @@ The js-ceramic node is run as a daemon using Docker or Node.js. By default, Cera
 
 **IPFS Out-of-process**
```

Comment:
Could we maybe include a link to this section above, in bullet 1 of the Getting Started section (https://github.com/ceramicnetwork/docs/pull/124/files#diff-b2a1dc0af145d769db9f7a6f3796521c5832641003fc25c35ff8e2e6a9a9c2d4R14)?  There we tell users they have to make this decision without given them any context on what that means.